build/build_edge_deb.sh

```diff
@@ -30,13 +30,16 @@ SAVED_OPTIONS=$(set +o)
 set +x
 # Get the canonical file name of the current script
 THIS_SCRIPT=`readlink -f ${BASH_SOURCE[0]}`
- 
+
 PROGRAM=`basename $0`
 
 # XRT Version variables
 XRT_MAJOR_VERSION=2
-XRT_MINOR_VERSION=13
-RELEASE_VERSION=202210
+XRT_MINOR_VERSION=15
+RELEASE_VERSION=202310
```

Comment:
I have the feeling that these values are more general than the scope of this file and should come from a single-truth file.

Reply:
Hi @keryell, there many flows and maintaining single xrt version file is not working out for all flows as some use cmake and some flows use shell scripts. 
The variables in this script are needed for changelog file.